eng/pipelines/templates/Initialize_Build.yml

```diff
@@ -30,9 +30,15 @@ steps:
           $targetChannel = & dotnet msbuild $env:BUILD_REPOSITORY_LOCALPATH\build\config.props /v:m /nologo /t:GetVsTargetChannel
           $targetChannel = $targetChannel.Trim()
         }
+        if ([System.String]::IsNullOrEmpty($env:VsTargetChannelOverrideForTests) -eq $false) {
```

Comment:
If `VsTargetChannelOverrideForTests` pipeline variable is configured, then Apex/E2E tests will run on that channel instead of default `int.main` channel.